rigs/heft-node-rig/profiles/default/tsconfig-base.json

```diff
@@ -17,6 +17,9 @@
     "noEmitOnError": false,
     "allowUnreachableCode": false,
 
+    "incremental": true,
+    "tsBuildInfoFile": "../../../../../lib/_tsBuildInfo.json",
```

Comment:
We could still stick this in the `.heft` folder.

Reply:
Putting it in the `lib/` folder makes incremental compilation work with the rush cache without also having to include `.heft` in the rush cache. Also, when running in solution mode, all such files will be part of `lib/` because their paths are dynamically resolved by the tools.
That said, I can move it back, but then will want to include it in default rush-project.json.